Source/SwiftLintFramework/Models/Version.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  Version.swift
+//  SwiftLint
+//
+//  Created by HirayamaYuya on 2016/09/16.
+//  Copyright © 2016年 Realm. All rights reserved.
+//
+
+import Foundation
+
+public struct Version: CustomStringConvertible {
+
+    public static var current: Version {
+        get {
+            guard let version = NSBundle(identifier: "io.realm.SwiftLintFramework")?
+                .objectForInfoDictionaryKey("CFBundleShortVersionString") as? String else {
+                    fatalError()
+            }
+
+            return Version(versionString: version)
+        }
+    }
+
+    var major: Int
+    var minor: Int
+    var patch: Int
+
+    init(major: Int, minor: Int, patch: Int) {
```

Comment:
It's possible you might want to provide default values for `minor` and `patch` and maybe even `major`:

`init(major: Int, minor: Int = 0, patch: Int = 0)`